packages/react/src/create-component.ts

```diff
@@ -309,7 +309,8 @@ export const createComponent = <
       // element properties in a special bag to be set by the server-side
       // element renderer.
       if (
-        React.createElement.name === 'litPatchedCreateElement' &&
+        (React.createElement.name === 'litPatchedCreateElement' ||
```

Comment:
Why is the patched name not enough?

Reply:
I haven't dug super into it but my suspicion is that production build can mangle the function name. I think it'd be fine to remove the name check all-together here too in favor of just checking the global flag. I left it more so as a holdover for those who won't immediately be updating `@lit-labs/ssr-react` but maybe that's over thinking it.